ios/PluginTests/NewRelicCapacitorPluginTests.swift

```diff
@@ -449,15 +449,15 @@ class NewRelicCapacitorPluginTests: XCTestCase {
                                                          "endTime": Date().timeIntervalSince1970,
                                                          "failure": "BadURL"],
                                                success: { (result, call) in
-            XCTAssertNotNil(result)
```

Comment:
this unit test looked like a copy and paste oversight from `testNoticeHttpTransaction`. However the objective here seems to prove that a `CAPPluginCall` with bad params should indeed make `NewRelicCapacitorPlugin.noticeHttpTransaction` fail, not succeed. So I adjusted the test accordingly, i.e. expecting an error.